projects/laji/src/app/+project-form/results/common/result-map/result-map.component.ts

```diff
@@ -112,6 +112,62 @@ export class ResultMapComponent implements OnInit {
 
   toHtmlSelectElement = toHtmlSelectElement;
 
+  private gatheringCountMapData$: Observable<DataOptions> = of([this.getGatheringCounts$(), this.getGatheringCounts$(true)]).pipe(
```

Comment:
Can you move long definitions like this into the constructor? So above the constructor just declare `private gatheringCountMapData$: Observable<DataOptions>;` and move the definition to `this.gatheringCountMapData$ = ...` in the constructor